Crash when sending ParaTime transaction to consensus

Using  849aceb92a3efca1c877db32cda35a4e6b14a998 I'm getting a CLI crash, if submitting ParaTime tx to consensus by accident:

```
$ cat testtx2.json 
{
  "Body": "o2F2AWJhaaJic2mBomVub25jZQFsYWRkcmVzc19zcGVjoWlzaWduYXR1cmWhZ2VkMjU1MTlYIDXD8zVt2FNk/roDVLVFraEJ0b2zi/XWEmgX24xyz9aRY2ZlZaJjZ2FzAWZhbW91bnSCQEBkY2FsbKJkYm9keaJidG9VAMjQ9FnbOOXMMcp35m0sRFbcvrUCZmFtb3VudIJIDeC2s6dkAABAZm1ldGhvZHFhY2NvdW50cy5UcmFuc2Zlcg==",
  "AuthProofs": [
    {
      "signature": "mxOARaaRkHrHiUObGO2KbUXZhmbIS2h3hsRM+3Y/DzvMtbVdwcZm2EZ7oP7dR5XHKkrQnjMwZnkT0mdZUsLIBw=="
    }
  ]
}

$ oasis tx submit testtx2.json --network mainnet  --no-paratime
Broadcasting transaction...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xc3aa5f]

goroutine 1 [running]:
github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection.(*connection).Runtime(0xc000014278, 0x0?)
        /home/oa/go/pkg/mod/github.com/oasisprotocol/oasis-sdk/client-sdk/go@v0.4.0/connection/connection.go:65 +0x5f
github.com/oasisprotocol/cli/cmd/common.BroadcastTransaction({0x13f8f50, 0xc000120000}, 0x0?, {0x13f71a0, 0xc000014278}, {0xf74340?, 0xc0007b47b0}, {0x0, 0x0}, {0x0, ...})
        /home/oa/cli/cmd/common/transaction.go:369 +0x127
github.com/oasisprotocol/cli/cmd.glob..func40(0x1a9ee40?, {0xc000161400, 0x1, 0x4?})
        /home/oa/cli/cmd/tx.go:66 +0x359
github.com/spf13/cobra.(*Command).execute(0x1a9ee40, {0xc0001613c0, 0x4, 0x4})
        /home/oa/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:944 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x1a9e880)
        /home/oa/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/oa/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/oasisprotocol/cli/cmd.Execute(...)
        /home/oa/cli/cmd/root.go:36
main.main()
        /home/oa/cli/main.go:6 +0x25
```

--- a/cmd/common/transaction.go
+++ b/cmd/common/transaction.go
@@ -160,6 +160,10 @@ func SignParaTimeTransaction(
 	tx *types.Transaction,
 	txDetails *signature.TxDetails,
 ) (*types.UnverifiedTransaction, interface{}, error) {
+	if npa.ParaTime == nil {
+		return nil, nil, fmt.Errorf("no paratime configured for paratime transaction signing")
+	}
+
 	// Default to passed values and do online estimation when possible.
 	nonce := txNonce
 	tx.AuthInfo.Fee.Gas = txGasLimit
@@ -282,7 +286,9 @@ func PrintTransaction(npa *NPASelection, tx interface{}) {
 		ctx := context.Background()
 		ctx = context.WithValue(ctx, consensusPretty.ContextKeyTokenSymbol, npa.Network.Denomination.Symbol)
 		ctx = context.WithValue(ctx, consensusPretty.ContextKeyTokenValueExponent, npa.Network.Denomination.Decimals)
-		ctx = context.WithValue(ctx, config.ContextKeyParaTimeCfg, npa.ParaTime)
+		if npa.ParaTime != nil {
+			ctx = context.WithValue(ctx, config.ContextKeyParaTimeCfg, npa.ParaTime)
+		}
 		ctx = context.WithValue(ctx, signature.ContextKeySigContext, &sigCtx)
 		ctx = context.WithValue(ctx, types.ContextKeyAccountNames, GenAccountNames())
 
@@ -365,6 +371,10 @@ func BroadcastTransaction(
 		fmt.Printf("Transaction hash: %s\n", sigTx.Hash())
 	case *types.UnverifiedTransaction:
 		// ParaTime transaction.
+		if pt == nil {
+			cobra.CheckErr("no paratime configured for paratime transaction submission")
+		}
+
 		fmt.Printf("Broadcasting transaction...\n")
 		rawMeta, err := conn.Runtime(pt).SubmitTxRawMeta(ctx, sigTx)
 		cobra.CheckErr(err)

